akka-docs/rst/java/code/docs/http/javadsl/server/HttpServerExampleDocTest.java

```diff
@@ -141,7 +141,9 @@ public TerminationDirective onUpstreamFailure(Throwable cause, Context<HttpReque
                 Flow.of(HttpRequest.class)
                     .via(failureDetection)
                     .map(request -> {
-                      Source<ByteString, Object> bytes = request.entity().getDataBytes();
+                      @SuppressWarnings("unchecked")
+                      Source<ByteString, Object> bytes =
+                          (Source<ByteString, Object>)(Source<ByteString, ?>) request.entity().getDataBytes();
                       HttpEntity.Chunked entity = HttpEntities.create(ContentTypes.TEXT_PLAIN_UTF8, bytes);
```

Comment:
When accepting a source  yet care not for the materialized values anyway we take `Any`thing.
When exposing sources yet have nothing interesting to materialize we say it's `NotUsed`. 

Two sides of the same coin kind of :)